packages/triutils/src/Trilinos_Util_ReadHpc2Epetra.cpp

```diff
@@ -80,7 +80,7 @@ void Trilinos_Util_ReadHpc2Epetra_internal(
     exit(1);
   }
   int_type numGlobalEquations, total_nnz;
-#ifdef DEBUG
+#ifdef NDEBUG
```

Comment:
It's `ifndef NDEBUG`.  `assert` only exists if `NDEBUG` is _not_ defined.

https://en.cppreference.com/w/c/error/assert

Reply:
Ahh!
I'm trying to burn through this a bit too fast today.